cmd/collector/app/zipkin/http_handler_test.go

```diff
@@ -225,7 +226,7 @@ func TestFormatBadBody(t *testing.T) {
 	statusCode, resBodyStr, err := postBytes(server.URL+`/api/v1/spans`, []byte("not good"), createHeader("application/x-thrift"))
 	assert.NoError(t, err)
 	assert.EqualValues(t, http.StatusBadRequest, statusCode)
-	assert.EqualValues(t, "Unable to process request body: Unknown data type 111\n", resBodyStr)
+	assert.True(t, strings.HasPrefix(resBodyStr, "Unable to process request body:"))
```

Comment:
nit: there is `assert.Contains` as well.

Reply:
I was sure there was something like this, but I was looking for "Prefix" and "Starts". "Contains" works too.